### PR TITLE
Added branches header and changed flow

### PR DESCRIPTION
**Pull request by epintos**
## Summary
- Now the show page of a projects shows the "metrics"
- Added the branches header to the nav to show all branches of a project
- Move the edit and add links to the right
- Fixed a bug when showing projects

### CODE REVIEW

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 end
 
 Sidekiq.configure_server do |config|
-  config.redis = { size: 2, url: url }
+  config.redis = { size: 30, url: url }
```
Comment (epintos) [inline config/initializers/sidekiq.rb]: It was failing with the 2 there :/

File: app/controllers/branches_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class BranchesController < ApplicationController
+  def index
+    organization
+    branches
+  end
+
+  private
+
+  def project
+    @project = Project.find(params[:project_id])
```
Comment (mdesanti) [inline app/controllers/branches_controller.rb]: I would add memoization here, just in case

File: app/controllers/branches_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  def branches
+    @branches ||= project.branches.order('branches.default DESC')
+  end
+
+  def organization
+    @organization = Organization.find(params[:organization_id])
```
Comment (mdesanti) [inline app/controllers/branches_controller.rb]: I would add memoization here, just in case

File: app/views/layouts/_header.html.slim [hunk 4]
```diff
@@ -13,14 +13,21 @@ nav.navbar.navbar-default
...
+            li
+              = link_to @project.name.camelize, organization_project_path(@organization, @project)
+            li
+              = link_to 'Branches', organization_project_branches_path(@organization, @project)
+
+      ul.nav.navbar-nav.navbar-right
+        - if @organization.present? && @organization.persisted?
           li
-            = link_to "#{@organization.name} Projects", organization_projects_path(@organization)
+            = link_to 'Add Project', new_organization_project_path(@organization)
```
Comment (mdesanti) [inline app/views/layouts/_header.html.slim]: Shouldn't this go with i18n?

File: app/views/layouts/_header.html.slim [hunk 5]
```diff
@@ -13,14 +13,21 @@ nav.navbar.navbar-default
...
           li
-            = link_to 'New Project', new_organization_project_path(@organization)
-      ul.nav.navbar-nav.navbar-right
+            = link_to "Edit Organization", organization_projects_path(@organization)
         li
-          = link_to 'New Organization', new_organization_path
+          = link_to 'Add Organization', new_organization_path
```
Comment (mdesanti) [inline app/views/layouts/_header.html.slim]: Shouldn't this go with i18n?

File: app/controllers/branches_controller.rb (hunk 2 above)
Comment (epintos) [inline app/controllers/branches_controller.rb]: Yeah, I forgot..

File: app/views/layouts/_header.html.slim (hunk 5 above)
Comment (epintos) [inline app/views/layouts/_header.html.slim]: Haha all the view needs. I will change them

**Pull request merged**